Content.Shared/GameObjects/EntitySystems/SharedMoverSystem.cs

```diff
@@ -92,12 +92,12 @@ protected void UpdateKinematics(ITransformComponent transform, IMoverComponent m
                     return;
                 }
 
-                var total = walkDir * mover.CurrentWalkSpeed + sprintDir * mover.CurrentSprintSpeed;
+                var total = walkDir * mover.CurrentWalkSpeed + sprintDir * mover.CurrentSprintSpeed * 1/frameTime;
```

Comment:
missing brackets
(walkDir * mover.CurrentWalkSpeed + sprintDir * mover.CurrentSprintSpeed) * 1/frameTime
also why do we sum walk and sprint speed, shouldn't sprint speed just be the actual sprint speed thats higher than walk speed?

Reply:
subtick movement my guy.

Reply:
ah, I see.. not going to dive into that now
still missing brackets :)